explainer.md

```diff
@@ -215,6 +246,226 @@ third-party then signs either the exchanges in a package or the package as a
 whole using a certificate whose metadata reflects whichever property was
 reviewed for.
 
+## Signed Exchange Loading Sketch
+
+Signed exchanges fit into the loading stack between the prefetch cache and
+Service Workers, leading to a stack with the following layers:
+
+Network → HTTP/2 Push cache → HTTP Cache → prefetch cache → **signed exchange
+handling** → Service Workers → preload cache → memory/image cache → actual
```

Comment:
Does the preload cache might look equivalent to the memory/image cache to developers? If so, I should probably merge them in this list.